eng/scripts/Fetch-PackageList-Javadoc.ps1

```diff
@@ -0,0 +1,51 @@
+<#
+  .SYNOPSIS
+  Fetch the package list from javadoc jar and checkin to docs/metadata
+
+  .DESCRIPTION
+  The Verify-Links.ps1 script will check whether the files contain any broken links.
+
+  .PARAMETER JavaDocJarLocation
+  Specifies the file that contains a set of links to ignore when verifying.
+
+  .PARAMETER DocRepoLocation
+  Specifies the file that contains a set of links to ignore when verifying.
+
+  .PARAMETER ArtifactName
+  Specifies the file that contains a set of links to ignore when verifying.
+#>
+[CmdletBinding()]
+param (
+  [string] $JavaDocJarLocation = "",
+  [string] $DocRepoLocation = "",
+  [string] $ArtifactName = ""
+)
+. (Join-Path $PSScriptRoot ../common/scripts/common.ps1)
+
+function Fetch-Namespaces-From-Javadoc ($jarFilePath, $tempLocation, $destination) {
+  Push-Location $tempLocation
```

Comment:
Instead of `Push-Location` we've been using something like: 

```pwsh 
$originalLocation = Get-Location 

try { 
  # Script logic...
} finally { 
  Set-Location $originalLocation
}
```

This ensures that an error in the script doesn't leave the user's terminal stuck in an unfamiliar location.